libs/pages/environments/src/lib/ui/container/container.tsx

```diff
@@ -24,6 +25,10 @@ export function Container(props: ContainerProps) {
     </div>
   )
 
+  const isDeploymentRulesTab =
+    location.pathname === `${ENVIRONMENTS_URL(organizationId, projectId)}${ENVIRONMENTS_DEPLOYMENT_RULES_URL}` ||
```

Comment:
is it possible to use matchPath to uniformise the way we do this kind of test?

` const matchLogInfraRoute = matchPath(pathname, INFRA_LOGS_URL(organizationId, clusterId))`